src/patternfly/demos/Page/page-template-title.hbs

```diff
@@ -1,3 +1,38 @@
+{{#> page-main-nav page-main-section--modifier="pf-m-light"}}
```

Comment:
Are you sure you want to add tertiary nav to every demo? Also the expandable nav already had tertiary nav, so now you have 2 in that demo - https://deploy-preview-1615--pf-next.netlify.com/demos/Page/examples-full/?component=Page%20Component%20Expandable%20Nav%20Example